src/code/providers/lara-provider.js

```diff
@@ -495,6 +495,8 @@ class LaraProvider extends ProviderInterface {
           return $.ajax({
             type: method,
             dataType: 'json',
+            contentType: 'application/json', // Document Store requires JSON currently
+            processData: false, // https://api.jquery.com/jquery.ajax/
             data: dataJson,
```

Comment:
Wouldn't it be worthwhile to compress this data just like the other upload code is doing?

Reply:
Sorry, I forgot about that.

b050dec re-adds the compression.  


Also I had to read the pako documentation, and then re-test everything, because I thought for sure `pako.deflate()` would decompress content.